asyncio/core.py

```diff
@@ -24,17 +24,17 @@
 except:
     from .task import TaskQueue, Task
 
+# Depending on the version of CircuitPython, these errors may exist in the build-in C code
+# even if _asyncio exists
```

Comment:
I'm confused. Do you mean "may not exist"? Also `build-in` -> `built-in`.

Reply:
Updated with new text, any better?

```
# Depending on the release of CircuitPython these errors may or may not
# exist in the C implementation of `_asyncio`.  However, when they
# do exist, must be preferred over the Python code.
```

Reply:
"they must be", but yes, that's good!

Reply:
updated, thanks!